generate_file_section.py

```diff
@@ -0,0 +1,290 @@
+import argparse
+import csv
+import fnmatch
+import os
+import re
+from pathlib import Path
+from pprint import pformat
+from warnings import warn
+
+
+def delete_commonpath(longer_path, prefix):
+    """return string with deleted common path."""
+
+    common_path = os.path.commonpath((longer_path, prefix)).split('/')
+    return '/'.join(longer_path.split('/')[len(common_path):])
+
+
+def locate_record(root, python3_sitelib, python3_sitearch):
+    """return path to record stripped of root path."""
+
+    record_path = list((Path(root) / Path(python3_sitelib[1:])).glob('*.dist-info/RECORD'))
+    record_path.extend(list((Path(root) / Path(python3_sitearch[1:])).glob('*.dist-info/RECORD')))
+
+    if len(record_path) == 0:
+        raise FileNotFoundError("Did not find RECORD file")
+    if len(record_path) > 1:
+        raise FileExistsError("Multiple *.dist-info directories")
+
+    record_path = str(record_path[0])
+    return "/" + delete_commonpath(record_path, root)
+
+
+def read_record(root, record_path):
+    """return parsed list [[[path], [hash], [size]], ...]"""
```

Comment:
why is it `[path]` and not just `path`, the same for others? i.e. why not:

```suggestion
    """returns parsed list of triplets like: [(path, hash, size), ...]"""
```

Note that lists are OK as well, if that's what csv reader gives you:
```suggestion
    """returns parsed list of triplets-like lists: [[path, hash, size], ...]"""
```